tox_poetry_installer/constants.py

```diff
@@ -20,7 +20,7 @@
 REPORTER_PREFIX: str = f"{__about__.__title__}:"
 
 # Internal list of packages that poetry has deemed unsafe and are excluded from the lockfile
-UNSAFE_PACKAGES: Set[str] = {"distribute", "pip", "setuptools", "wheel"}
+UNSAFE_PACKAGES: Set[str] = set()
```

Comment:
Can you add a todo comment here noting that the functionality that uses this constant should be removed in a later update, now that it's no longer needed?